acceptance_tests/features/pages/inbox.py

```diff
@@ -0,0 +1,54 @@
+
+from datetime import datetime
+from config import Config
+
+from acceptance_tests import browser
+
+
+def go_to():
+    browser.visit(f"{Config.RESPONSE_OPERATIONS_UI}/messages")
+
+
+def get_page_title():
+    return browser.title
+
+
+def get_messages():
+    messages = []
+    table = browser.find_by_id('tbl-messages')
+    rows = table.find_by_tag('tbody').find_by_tag('tr')
+    for row in rows:
+        messages.append({
+            'ru_ref': row.find_by_name('tbl-messages-RU_Ref').value,
+            'business_name': row.find_by_name('tbl-messages-business').value,
+            'subject': row.find_by_name('tbl-messages-subject').value,
+            'from': row.find_by_name('tbl-messages-from').value,
+            'to': row.find_by_name('tbl-messages-to').value,
+            'received': row.find_by_name('tbl-messages-received').value
+        })
+    return messages
+
+
+def are_headings_correct():
```

Comment:
Maybe get_headings and asserting the headings in the test will make the test more readable